test_pull_requests

```diff
@@ -124,12 +124,44 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
+CONTENT_WAITING_IN_QUEUE = "Waiting: You are in the build queue at position:"
+CONTENT_WAITING_DETERMINING_QUEUE_POS = "Waiting: Determining build queue position"
+CONTENT_WAITING_STABLE_BUILD = "Waiting for stable build of"
+CONTENT_RUNNING = "Running"
+CONTENT_WAITING = "Waiting"
+CONTENT_EVALUATING = "Evaluating for testing"
+
+CONTENT_TO_STATE = {
+  CONTENT_EVALUATING => :evaluating,
+  CONTENT_RUNNING    => :running,
+  CONTENT_WAITING    => :waiting,
```

Comment:
So in those cases you're using the reverse mapping to get the "content" but then ignoring it to post your own content anyway? Seems shaky at best -- I'm not sure what ruby's `invert` does on a map that's not one-to-one.

In a perfect world the refactor to remove the aliasing on `:waiting` would be a pre-req to building a mapping such as we have now, I guess. Yuck yuck yuck